src/redux/weather/weatherSlice.js

```diff
@@ -0,0 +1,35 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+const FETCH = 'weather/FETCH';
+
+const initialState = {
+  weather: [],
+};
+
+export const getWeatherFromRandomCities = createAsyncThunk(FETCH, async () => {
+  const weather = [{
+    id: 1, name: 'london', country: 'United Kingdom', localtime: '12:09 PM', humidity: '80%', temp_c: '23',
+  }, {
+    id: 2, name: 'paris', country: 'France', localtime: '1:09 PM', humidity: '10%', temp_c: '12',
+  }, {
+    id: 3, name: 'kinshasa', country: 'DR Congo', localtime: '1:09 PM', humidity: '27%', temp_c: '32',
+  }, {
+    id: 4, name: 'san francisco', country: 'United States', localtime: '8:09 PM', humidity: '18%', temp_c: '15',
+  }];
+  return weather;
+});
```

Comment:
- [ ] You're not interacting with an external API as required by the project, please don't seed static data to your app like this, you're not meeting one of the most important goals of this activity which is to consume APIs in your React/Redux apps. 🤓 🔧